example/example.ipynb

```diff
@@ -533,6 +533,73 @@
     "counts = retrieved.result().get_counts()\n",
     "plot_histogram(counts)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Submitting multiple circuits in parallel \n",
+    "\n",
+    "We can use [multiprocessing](https://docs.python.org/3/library/multiprocessing.html) to submit multiple jobs containing different circuits to IonQ backend. Getting all results at once without blocking prior to all jobs being submitted."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "s:  0\n",
+      "s:  1\n",
+      "s:  10\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Create multiple circuits using previously defined Bernstein-Vazirani circuit\n",
+    "circuits = []\n",
+    "for i in range(3):\n",
+    "    circuit = make_bv_circuit(i, i)\n",
+    "    circuits.append(circuit)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 36,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import time\n",
+    "\n",
+    "def get_result(bv_job):\n",
```

Comment:
This takes in a job, but is passed a circuit below.

You will need to call backend.run() here (or below before passing the job object into the threadpool, whichever works).